doc/rtd/reference/distros.rst

```diff
@@ -0,0 +1,35 @@
+Supported distros
```

Comment:
This needs a brief explanation to say what this page is for. Although we know it's linked from the readme, users would be able to find it just in the navigation menu so we can't assume that's how they found it - we need to tell them the purpose of this list/page, and also add a comment at the bottom to point out where/how to add a distro

Reply:
Done

Reply:
Looks good, thanks!